### PR TITLE
docs: Keboola integration doc

### PR DESCRIPTION
**Pull request by TC-MO**
create new Keboola integration doc
add new screenshots

### CODE REVIEW

File: sources/platform/integrations/keboola.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+description: Learn how to integrate your Apify datasets with Airbyte.
+sidebar_position: 11.16
+slug: /integrations/keboola
+---
+
+**Integrate your Apify Actors with Keboola.**
+
+---
+
+With Apify integration for [Keboola](https://www.keboola.com/), you can extract data from various sources using your Apify Actors and load it into Keboola for further processing, transformation, and integration with other platforms.
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:13]:
```suggestion
With the Apify integration for [Keboola](https://www.keboola.com/), you can extract data from various sources using your Apify Actors and load it into Keboola for further processing, transformation, and integration with other platforms.
```

File: sources/platform/integrations/keboola.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+The Keboola integration allows you to run your Actors, fetch items from datasets, and retrieve results, all within the Keboola platform.
+
+## Connect Apify with Keboola
+
+To use the Apify integration on Keboola, you will need to:
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:19]:
```suggestion
To use the Apify integration on Keboola, you will need:
```

File: sources/platform/integrations/keboola.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+- Have an [Apify account](https://console.apify.com/).
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:21]:
```suggestion
- An [Apify account](https://console.apify.com/).
```

File: sources/platform/integrations/keboola.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+- Have a [Keboola account](https://www.keboola.com/).
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:22]:
```suggestion
- A [Keboola account](https://www.keboola.com/).
```

File: sources/platform/integrations/keboola.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+### Step 1: Create a New Data Source in Keboola
+
+Once your Keboola account is ready and you are logged in, navigate to the **Components** section in the top menu and click the **Add Component** button.
+
+![Keboola dashboard](./images/keboola/keboola-dashboard.png)
+
+In the list of available Components, find and select the **Apify** from Data Sources and click on the **Add Component** button.
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:30]:
```suggestion
In the list of available Components, find and select the **Apify** option from Data Sources and click on the **Add Component** button.
```

File: sources/platform/integrations/keboola.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+![Keboola configuration setup](./images/keboola/keboola-create-configuration.png)
+
+### Step 2: Configure the Apify Data Source
+
+With the new configuration created, you can now configure the data source to retrieve the needed data. Click on the **Configure Component** button to begin the setup process.
+
+![Keboola component configuration](./images/keboola/keboola-configure-component.png)
+
+
+#### Choose an Action
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:49]:
```suggestion
#### Choose an action
```

File: sources/platform/integrations/keboola.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+The Keboola integration allows you to run your Actors, fetch items from datasets, and retrieve results, all within the Keboola platform.
+
+## Connect Apify with Keboola
+
+To use the Apify integration on Keboola, you will need to:
+
+- Have an [Apify account](https://console.apify.com/).
+- Have a [Keboola account](https://www.keboola.com/).
+
+### Step 1: Create a New Data Source in Keboola
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:24]:
```suggestion
### Step 1: Create a new data source in Keboola
```

File: sources/platform/integrations/keboola.md (hunk 5 above)
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:30]:
```suggestion
In the list of available components, find and select the **Apify** from Data Sources and click on the **Add Component** button.
```

File: sources/platform/integrations/keboola.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+On the new window, click **Connect** to my data**
+
+![Keboola Connector](./images/keboola/keboola-connector.png)
+
+Provide a name and description for your configuration, and then click on the **Create Configuration** button.
+
+![Keboola configuration setup](./images/keboola/keboola-create-configuration.png)
+
+### Step 2: Configure the Apify Data Source
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:42]:
```suggestion
### Step 2: Configure the Apify data source
```

File: sources/platform/integrations/keboola.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+- **Output field**:  Comma-separated list of fields to be picked from the dataset.
+- **Memory**: Adjust the memory settings if needed (the default values can be kept).
+- **Build**: Adjust if you want to run a specific build of an Actor. Tag or number of the build to run.
+- **Actor Input**: Pass any JSON data as input to the Actor.
+
+Once you have filled in all the necessary options, click the **Save** button to save your configuration.
+
+![Keboola component specification setup](./images/keboola/keboola-setup-specification.png)
+
+### Step 4: Run the Configured Data Source
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:82]:
```suggestion
### Step 4: Run the configured data source
```

File: sources/platform/integrations/keboola.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+After your data source has been configured, you can run it by clicking the **Run** button in the upper-right corner of your configuration.
+
+![Keboola run configured component](./images/keboola/keboola-run-component.png)
+
+You can monitor the progress of your run in the job detail section on the right-hand side of the page.
+
+Once the run finishes successfully, you can find the results by following the link in the Storage Stats section of the job detail page.
+
+## Next Steps
```
Comment (TheoVasilis) [inline sources/platform/integrations/keboola.md:92]:
```suggestion
## Next steps
```

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

File: sources/platform/integrations/keboola.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
+---
+title: Keboola
+sidebar_lbel: Keboola
+description: Learn how to integrate your Apify datasets with Airbyte.
+sidebar_position: 11.16
+slug: /integrations/keboola
+---
+
+**Integrate your Apify Actors with Keboola.**
```
Comment (drobnikj) [inline sources/platform/integrations/keboola.md:9]: I would describe Keboola itself, how it work and for what you can use it.
You can reuse some text from https://help.apify.com/en/articles/2003234-keboola-integration

File: sources/platform/integrations/keboola.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+
+You can monitor the progress of your run in the job detail section on the right-hand side of the page.
+
+Once the run finishes successfully, you can find the results by following the link in the Storage Stats section of the job detail page.
+
+## Next Steps
+
+With your data now in Keboola, you can integrate it with dozens of other services that Keboola supports. Check out the [full list of integrations](https://www.keboola.com/integrations) to explore your options.
+
+You can set up a writer for a selected service using Keboola Writer or create orchestrations to transform, merge, or split your data as needed.
```
Comment (drobnikj) [inline sources/platform/integrations/keboola.md:96]: Let's put the link about Keboola orchestration here.

File: sources/platform/integrations/keboola.md (hunk 10 above)
Comment (drobnikj) [inline sources/platform/integrations/keboola.md:92]: I'm not sure, where, but we should mention this Actor in store.
https://apify.com/drobnikj/keboola-input-mapping
It is helper to start Actor run with data from Keboola.

Plus once you finish this docs, we need to link the doc from the Actor. You can simply create PR to https://github.com/drobnikj/keboola-ex-input-table-mapping

**Review by drobnikj — CHANGES_REQUESTED** [review]
A few ideas for improvements

**Comment by TC-MO** [comment]
@TheoVasilis question, Keboola themselves refer to Components & Data Sources with capitalization, should we follow this convention since this seems to be their proper name for these features?

File: sources/platform/integrations/keboola.md (hunk 12 above)
Comment (TC-MO) [reply sources/platform/integrations/keboola.md]: added

File: sources/platform/integrations/keboola.md (hunk 11 above)
Comment (TC-MO) [reply sources/platform/integrations/keboola.md]: expanded the intro hook slightly

File: sources/platform/integrations/keboola.md (hunk 10 above)
Comment (TC-MO) [reply sources/platform/integrations/keboola.md:92]: added a note regarding the helper actor

**Review by TheoVasilis — APPROVED** [review]

**Review by drobnikj — APPROVED** [review]

**Pull request merged**